plugins/auth/src/auth-oidc/index.js

```diff
@@ -1,18 +1,17 @@
-import { UserManager, WebStorageStateStore, InMemoryWebStorage } from 'oidc-client';
+import { UserManager, WebStorageStateStore, InMemoryWebStorage } from 'oidc-client-ts';
 import { Helpers } from '../helpers';
 import { thirdPartyCookiesStatus } from '../third-party-cookies-check';
+
 export default class openIdConnect {
   constructor(settings = {}) {
     const defaultSettings = {
-      redirect_uri: window.location.origin,
-      post_logout_redirect_uri: window.location.origin + '/logout.html',
-      response_type: 'id_token token',
+      accessTokenExpiringNotificationTimeInSeconds: 60,
+      automaticSilentRenew: false,
       filterProtocolClaims: true,
       loadUserInfo: false,
-      automaticSilentRenew: false,
-      accessTokenExpiringNotificationTime: 60,
-      thirdPartyCookiesScriptLocation: '',
-      logoutUrl: window.location.origin + '/logout.html',
+      post_logout_redirect_uri: window.location.origin + '/logout.html',
+      redirect_uri: window.location.origin,
+      response_type: 'code',
       silent_redirect_uri: window.location.origin + '/assets/auth-oidc/silent-callback.html'
     };
 
```

Comment:
Logout does not work for me. Getting the following error:
```
extendedConfiguration.bundle.js:2 Uncaught (in promise) TypeError: this.client.createSignoutRequest is not a function
    at e.value (extendedConfiguration.bundle.js:2:78458)
    at K2.logout (auth-layer.js:146:32)
    at pe (AuthorizationSimpleProfileMenu.svelte:180:18)
    at AuthorizationSimpleProfileMenu.svelte:163:9
   ```
_createSignoutRequest_ does not exist on UserManager (anymore)
Line 90

Reply:
Let me check what's going on

Reply:
There was small change for 'UserManager' API in 'oidc-client-ts' - I fixed it